README.md

```diff
@@ -1,21 +1,45 @@
 # BPMN Visu JS
-A JavaScript library to visualize process execution data. Diagrams are displayed from [BPMN](https://www.omg.org/spec/BPMN/2.0.2/PDF) files. With additional display
-options for execution data (highlight some transitions, counters, and more). With interactive capacities (mouse hover,
-click).
+A JavaScript library to visualize [BPMN](https://www.omg.org/spec/BPMN/2.0.2/PDF) diagrams as well as meta-information about process execution, which the user can interact with.
+  - BPMN diagrams are read from .bpmn files that may not only come from Bonita but rather be created from any BPM tool or even created manually. This means that it will be able to display more BPMN 2.0 elements than those supported by Bonita.
+  - Process execution information can be real-time or over a period of time in the past. It encompasses information on cardinality (counters), relative frequencies (color density, transition thickness) or durations. 
+  - Interactivity is brought by the means of mouse hover, clicks, and runtime configuration options to define.
+
+This library will be created and offered as an Open Source project.
```

Comment:
I'm not sure this information is relevant for a user of the library. The repository is already Public, and we have a license who indicates what the user can do with it.